src/services/wishlist/addToWishlist.js

```diff
@@ -0,0 +1,11 @@
+import axios from "axios";
+
+const addToWishlist = (product, token) => {
+  return axios.post(
+    "/api/user/wishlist/",
+    { product },
+    { headers: { authorization: token } }
+  );
+};
```

Comment:
instead of writing every function in different files and exporting them from there you could have made one file and exported all of them from there, this could have made your code easier to understand